src/index.js

```diff
@@ -57,7 +57,7 @@ export function app(state, actions, view, container) {
 
   function resolveNode(node) {
     return typeof node === "function"
-      ? resolveNode(node(globalState, wiredActions))
+      ? resolveNode(node(globalState, wiredActions)) || node
```

Comment:
When `node` is a function:
1. `node(gstate, gactions)` => `null`
2. `resolveNode(null)` => `null`
3. `null || function` => `function`

`function` is not a valid vnode type